tests/integration_tests/test_execution.py

```diff
@@ -0,0 +1,598 @@
+            privacy_request,
+            policy,
+            mongo_postgres_dataset_graph,
+            [integration_postgres_config, integration_mongodb_config],
+            {"email": "customer-1@example.com"},
+        )
+
+        execution_logs = get_sorted_execution_logs(db, privacy_request)
+        assert execution_logs == [
+            ("postgres_example_test_dataset:customer", "in_processing"),
+            ("postgres_example_test_dataset:customer", "complete"),
+            ("postgres_example_test_dataset:payment_card", "in_processing"),
+            ("postgres_example_test_dataset:payment_card", "complete"),
+            ("postgres_example_test_dataset:orders", "in_processing"),
+            ("postgres_example_test_dataset:orders", "complete"),
+            ("postgres_example_test_dataset:order_item", "in_processing"),
+            ("postgres_example_test_dataset:order_item", "complete"),
+            ("postgres_example_test_dataset:product", "in_processing"),
+            ("postgres_example_test_dataset:product", "complete"),
+            ("mongo_test:customer_details", "in_processing"),
+            ("mongo_test:customer_details", "error"),
+            ("postgres_example_test_dataset:employee", "in_processing"),
+            ("postgres_example_test_dataset:employee", "complete"),
+            ("postgres_example_test_dataset:service_request", "in_processing"),
+            ("postgres_example_test_dataset:service_request", "complete"),
+            ("mongo_test:customer_feedback", "skipped"),
+            ("mongo_test:internal_customer_profile", "skipped"),
+            ("mongo_test:rewards", "skipped"),
+            ("postgres_example_test_dataset:report", "in_processing"),
+            ("postgres_example_test_dataset:report", "complete"),
+            ("postgres_example_test_dataset:visit", "in_processing"),
+            ("postgres_example_test_dataset:visit", "complete"),
+            ("postgres_example_test_dataset:address", "in_processing"),
+            ("postgres_example_test_dataset:address", "complete"),
+            ("mongo_test:customer_details", "skipped"),
+            ("mongo_test:flights", "skipped"),
+            ("mongo_test:employee", "skipped"),
+            ("mongo_test:aircraft", "skipped"),
+            ("mongo_test:conversations", "skipped"),
+            ("mongo_test:payment_card", "skipped"),
+            ("postgres_example_test_dataset:login", "in_processing"),
+            ("postgres_example_test_dataset:login", "complete"),
+        ], "Rerun skips disabled collections"
+
+        assert all(
+            [dataset.startswith("postgres_example") for dataset in results]
+        ), "No mongo results"
+
+
+@pytest.mark.integration
+def test_restart_graph_from_failure(
+    db,
+    policy,
+    example_datasets,
+    integration_postgres_config,
+    integration_mongodb_config,
+    mongo_postgres_dataset_graph,
+) -> None:
+    """Run a failed privacy request and restart from failure"""
+
```

Comment:
This test isn't new, I've just renamed the file and added more execution-related tests